joomla/joomla-cms
# [com_installer] Add search tooltip to all views

#### Summary of Changes

Simple PR to improve the search tooltip and element label (accessibility) in com_installer.
#### Testing Instructions

Code review, or:
1. Apply patch
2. Go to Extensions -> Manage -> Update, check the tooltip in the search field.
3. Go to Extensions -> Manage -> Manage, check the tooltip in the search field.
4. Go to Extensions -> Manage -> Discover, check the tooltip in the search field.
5. Go to Extensions -> Manage -> Install Languages, check the tooltip in the search field.
6. Go to Extensions -> Manage -> Update Sites, check the tooltip in the search field.
